src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -307,25 +307,23 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_OFI_win_request_complete(MPIDI_OFI_win_reque
     }
 }
 
-MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_comm_to_phys(MPIR_Comm * comm, int rank, int ep_family)
```

Comment:
It feels like extra index/ep number instead of family would be more universal solution
For instance we could have two types of functions:

`MPIDI_OFI_comm_to_phys(MPIR_Comm * comm, int rank)`
`MPIDI_OFI_comm_to_phys_ep(MPIR_Comm * comm, int rank, int ep_num)`

The second function would give us more freedom. The first one would use some default ep selection logic.